cr-interactors/src/main/java/com/thefuntasty/mvvm/crinteractors/BaseCrViewModel.kt

```diff
@@ -7,8 +7,8 @@ import kotlinx.coroutines.CoroutineScope
 
 /**
  * Base ViewModel class prepared for providing data to UI through [LiveData] and
- * obtaining data from Stores (Repositories) by executing Coroutine based interactors like
- * [BaseCoroutineInteractor] and [BaseFlowInteractor].
+ * obtaining data from Stores (Repositories) by executing Coroutine based use cases like
+ * [BaseUseCase] and [BaseFlowUseCase].
```

Comment:
Akorát když se tohle přejmenu ja UseCase tak nedává smysl aby se ten modul jmenoval cr-interactors

Reply:
Yep, all change it simultaneously with rx-interactors within task #81